privaterelay/templates/profile.html

```diff
@@ -18,6 +18,13 @@
   data-fxa-subscriptions-url="{{ settings.FXA_SUBSCRIPTIONS_URL }}" 
   data-premium-prod-id="{{ settings.PREMIUM_PROD_ID }}" 
   data-premium-price-id="{{ settings.PREMIUM_PRICE_ID }}" 
+  {% if settings.PREMIUM_ENABLED %}
+    {% with request.user.profile_set.first as user_profile %}
+      {% if user_profile.subdomain %}
```

Comment:
Is there a better way to chain this than three if/with type statements?

Reply:
Hmm ... we can probably add this markup to the page even if `PREMIUM_ENABLED` is `False`? It won't hurt anything, right?

We should also make our own context processor that always puts a `user_profile` variable in every template context.